### ISSUE TITLE
What's this?

### ISSUE THREAD

**Issue by davidak** [gytis-ivaskevicius/linux-kernel-comparison#1]
Can you add a README and a LICENSE for it to be useful to people?

Could this help compare kernel options of nixos and other distros to find out why this issue exist? https://github.com/NixOS/nixpkgs/issues/67113

**Comment by gytis-ivaskevicius** [comment]
Yeah, this little 'flake/devshell powered' tool helps you to do a couple of things:
- [Fetch different kernel configs ](https://github.com/gytis-ivaskevicius/linux-kernel-comparison/blob/master/flake.nix#L27-L32)
- [Normalize/Cleanup/Deduplicate fetched configs](https://github.com/gytis-ivaskevicius/linux-kernel-comparison/blob/master/flake.nix#L35-L50)
- [Shows the differences between fetched configs](https://github.com/gytis-ivaskevicius/linux-kernel-comparison/blob/master/flake.nix#L55-L64) 
- [Also it shows you documentation of the option that you are currently comparing (powered by terminal browser lynx :D)](https://github.com/gytis-ivaskevicius/linux-kernel-comparison/blob/master/flake.nix#L58-L60)

At some point, I was going to investigate what's going on with the NixOS kernel but never ended up finishing this adventure

This weekend I could check the performance of my CPU and see if I can reproduce issues that you are having (I got Ryzen 5950x)
And I guess while I'm at it - I might as well go over the options, tho ill probably drop Arch config since if I recall correctly it has a lot of `CONFIG_XYZ=m` to support a bunch of hardware

**Comment by gytis-ivaskevicius** [comment]
If you are curious you can see how much configs changed since the last time I touched it https://github.com/gytis-ivaskevicius/linux-kernel-comparison/pull/2/files
